### DISCUSSION TITLE
How do I avoid caching subpaths?

### DISCUSSION THREAD

**Discussion by clearbluejar** [cotes2020/jekyll-theme-chirpy#1424] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Mostly the same question here, I just haven't figured out how to "uncache" subpaths.

I mostly want my other github page subpaths to avoid caching (they are subpaths of this main site). 

https://github.com/cotes2020/jekyll-theme-chirpy/issues/508#issue-1139699221

### Operations you have already tried

I have tried adding some to my config, but I don't quite get how it works. 

### Anything else?

https://github.com/clearbluejar/clearbluejar.github.io/blob/main/_config.yml

**Comment by cotes2020** [comment]
> - #508 

Ah, looks like I missed an old issue. 



Adding a configuration file for PWA's `denyUrls` could be a candidate for a new feature.

Personally, I'd prefer to add the config file under `_data/` rather than `_config.yml`, so that `_config.yml` doesn't get overstuffed as the project grows.

**Reply by clearbluejar** [reply]
I don't have a preference for config or _data, just the ability as it affects my subpaths. I misunderstood and thought it was implemented and I just couldn't figure it out. Is it OK to reopen the old issue?

**Reply by cotes2020** [reply]
> Is it OK to reopen the old issue?

Shouldn't have to, that issue was poorly described and at the time I skipped it because I didn't know what he was trying to say.

**Reply by cotes2020** [reply]
- Converted to #1454

**Comment by cotes2020** [comment]
Hi @clearbluejar, I'd like to know what exactly you mean by "subpath". A specific list would be appreciated, please.

**Reply by clearbluejar** [reply]
Hi. The issue occurs because I use subpaths of my github.io domain for different websites cache according the rules setup under the primary site. 

- https://clearbluejar.github.io/   main blog site with chirpy, caches and updates when changes occur. 
- https://clearbluejar.github.io/ghidriff another web site hosted on github pages for a completely different purpose. 

This issue is that all the pages under https://clearbluejar.github.io/ghidriff will be cached according the caching of the main blog site (and there is no "update" feature for the subpages). 

I want to avoid caching specific subpaths:
- https://clearbluejar.github.io/ghidriff/*
- https://clearbluejar.github.io/anothersite_github_site/*

**Reply by cotes2020** [reply]
Thanks for adding the details, that makes sense.
